pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -444,17 +444,22 @@ func (ca *changeAggregator) Next() (rowenc.EncDatumRow, *execinfrapb.ProducerMet
 			return ca.ProcessRowHelper(ca.resolvedSpanBuf.Pop()), nil
 		}
 		if err := ca.tick(); err != nil {
-			if errors.Is(err, kvevent.ErrBufferClosed) {
+			var e kvevent.ErrBufferClosed
+			if errors.As(err, &e) {
```

Comment:
I think that `errors.Is` knows how to walk up this unwrap chain:

https://github.com/cockroachdb/errors/blob/72ec75208141a2b67e79909b28aea29ed2189108/markers_api.go#L35
https://github.com/cockroachdb/errors/blob/72ec75208141a2b67e79909b28aea29ed2189108/markers/markers.go#L51-L54

so you may be able to simplify some of this.

Reply:
I need to check both levels of error, so I can't simplify it too much, but I've added an `Is` below.